tools/cbpManager.deploy/shinyproxy/application.yml

```diff
@@ -0,0 +1,31 @@
+proxy:
```

Comment:
Here same thing as with our .env file that we had in the project root.

If you expect users to configure their own IDP here, changes to this file will cause merge conflicts.

Reply:
In contrast to the .env file I whitelisted this file, because it is obligatory to run shinyproxy.